src/quo/components/wallet/account_overview/style.cljs

```diff
@@ -3,12 +3,9 @@
     [quo.foundations.colors :as colors]))
 
 (def account-overview-wrapper
-  {:padding-top        24
-   :padding-horizontal 20
-   :padding-bottom     20
-   :height             130
-   :align-items        :center
-   :justify-content    :center})
+  {:height          110
```

Comment:
moved this to the page using and passed via container-style - (also updated preview screen)